recipes/pymssql/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pymssql" %}
+{% set version = "2.2.0pre1" %}
```

Comment:
do we know that 2.2.0 is going to be the next release?

Reply:
If not this should be 2.1.3.post# where # is how many commits after 2.1.3 `c3b445b2f7dd1b9d1ee8028879795f7671a8f86f` is